rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java

```diff
@@ -147,6 +147,15 @@
 @ActiveObject
 public class RMCore implements ResourceManager, InitActive, RunActive {
 
+    /** Limits the number of nodes the Resource Manager accepts. >-1 means UNLIMITED, <0 enforces the limit.
```

Comment:
Based on the implementation, the comment seems wrong. `<= -1` is unlimited and `>0` enforces the limit?